src/vm/types/mod.rs

```diff
@@ -35,16 +35,72 @@ pub struct ListData {
     pub type_signature: ListTypeData
 }
 
-// a standard principal is a version byte + hash160 (20 bytes)
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
 pub struct StandardPrincipalData(pub u8, pub [u8; 20]);
 
+impl StandardPrincipalData {
+
+    pub fn transient() -> Self {
```

Comment:
Is this used anywhere outside of testing?  If not, then it should be decorated with `#[cfg(test)]`

Reply:
It's also being used here and there in `clarity.rs`:

https://github.com/blockstack/blockstack-core/blob/fd249be6683ab9f81cea89c31c9a96a1290dc430/src/clarity.rs#L120-L131

Reply:
Okay, then we can leave as-is :+1: